ppr-ui/src/composables/useUpdatedBadges.ts

```diff
@@ -0,0 +1,17 @@
+import { ComputedRef, computed } from 'vue'
+import { useMhrCorrections } from './mhrRegistration'
+import { useStore } from '@/store/store'
+import { storeToRefs } from 'pinia'
+
+export const useUpdatedBadges = () => {
+
+  const { isMhrReRegistration } = storeToRefs(useStore())
+  const { isMhrCorrection } = useMhrCorrections()
+
+  const showUpdatedBadge: ComputedRef<boolean> = computed((): boolean =>
+    isMhrCorrection.value || isMhrReRegistration.value)
+
+  return {
+    showUpdatedBadge
+  }
+}
```

Comment:
I'm not sure we need a composable for a single computed value, this would probably make sense to include in useMhrCorrection

Reply:
The idea was to capture all scenarios where badge needs to be shown, have them separately without mixing. Yea it's just one computed at the moment, but we can refactor and bring in other things, like baseline/current states based on flow, etc. If no interest, and in foreseeable future we can get away with sharing logic in other composables, im totally ok to reverse it.

Reply:
If you think it will serve a future purpose, than all good!